pkg/features/kube_features.go

```diff
@@ -217,22 +177,6 @@ const (
 	// NodePublishVolume calls.
 	DelegateFSGroupToCSIDriver featuregate.Feature = "DelegateFSGroupToCSIDriver"
```

Comment:
Same as above.

Reply:
Same as `CSIMigrationAzureFile `, It should be removed in 1.28, So maybe it should wait until 1.28 ?

Reply:
that's correct, `CSIMigrationAzureFile` should be removed in 1.28